cmd/loki-canary-test/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM golang:1.18.5 as build
```

Comment:
I'm not sure we want to publish this. As users should use alerts on canary metrics instead.

Reply:
We need to publish these tests in a way that is easily consumable by a k8s Pod spec, so I think publishing an image with the compiled test binary is probably the easiest way to do that.